src/kw_remote.sh

```diff
@@ -127,11 +169,11 @@ function add_new_remote()
   fi
 
   # Check if remote name already exists
-  grep -xq "^Host ${name}$" "$local_remote_config_file"
+  grep -xq "^Host ${name}$" "$remote_config_file"
```

Comment:
Since you are changing this file, can you add one extra commit where you replace:

`grep -xq ..` with `grep --line-regexp --quiet`.

Using the full parameter name in the bash code makes it easier to understand what's going on in the line.

P.s.: I'm not sure if we documented this idea, but if not we should.